liminal/util/fixTemplateStrings.ts

```diff
@@ -0,0 +1,14 @@
+const INDENTATION_RE = /^\n( +)/
```

Comment:
I suppose we could support `/([ \t]+)/` in case we have people using tabs to indent their template strings

Reply:
applied